### PR TITLE
Added

### PR DESCRIPTION
**Pull request by venuarelli**

### CODE REVIEW

File: src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java [hunk 1]
```diff
@@ -34,7 +34,6 @@ public void testMultiWindowHandling(){
 	}
 	
 	@Test
-	
 	public void testInnovapath(){
```
Comment (VenkiDeveloper) [inline src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java:37]: Please check method name

File: src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java [hunk 2]
```diff
@@ -34,7 +34,6 @@ public void testMultiWindowHandling(){
 	}
 	
 	@Test
```
Comment (VenkiDeveloper) [inline src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java:36]: Add test description

File: src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java (hunk 1 above)
Comment (venuarelli) [inline src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java:37]: done

File: src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java (hunk 2 above)
Comment (venuarelli) [inline src/com/apipracticeexamples/tests/MultiWindowHandlingTest.java:36]: done

**Pull request merged**